rom-hacking/scripting/animation-commands/ac-wait.md

```diff
@@ -0,0 +1,29 @@
+# (779) AC_WAIT
+
+## Effect
+
+Waits a given amount of time.
+
+Sets animation clip to 0 (Idle).
+
+## Syntax
+
+```c
+_AC_WAIT(time)
```

Comment:
Perhaps instead of using the word time, since it's determined in frames and other commands in this PR use actual time length, it would be best to use `frames` here.